src/Cards.py

```diff
@@ -1,4 +1,14 @@
 from pygame import draw
+
+colours = {
+    "red" : (255, 0, 0),
+    "blue" : (0, 0, 255),
+    "green" : (0, 255, 0),
+    "yellow" : (255, 255, 0),
+    "white" : (255, 255, 255),
+    "black" : (0, 0, 0)
+}
+
```

Comment:
We'll figure out better colours instead of straight up those colours. Nicer colours like

```py
colours = {
    ...
    "red": (255, 23, 68)
    ...
}
```